specs/data_structures.md

```diff
@@ -44,27 +44,24 @@ Blocks are the top-level data structure of the LazyLedger blockchain.
-| `time`               | [Time](#time)             | Timestamp of this block.                                           |
-| `lastBlockID`        | [BlockID](#blockid)       | Previous block's ID.                                               |
-| `lastCommitRoot`     | [HashDigest](#hashdigest) | Previous block's Tendermint commit root.                           |
-| `validatorsRoot`     | [HashDigest](#hashdigest) | Validator set root for this block.                                 |
-| `nextValidatorsRoot` | [HashDigest](#hashdigest) | Root of the next block's validator set.                            |
-| `consensusHash`      | [HashDigest](#hashdigest) | Consensus parameters for this block.                               |
-| `evidenceRoot`       | [HashDigest](#hashdigest) | Evidence data root.                                                |
-| `availableDataRoot`  | [HashDigest](#hashdigest) | Root of [commitments to erasure-coded data](#availabledataheader). |
-| `proposerAddress`    | [Address](#address)       | Address of this block's proposer.                                  |
+| name                | type                      | description                                                                                  |
+| ------------------- | ------------------------- | -------------------------------------------------------------------------------------------- |
+| `version`           | `uint64`                  | Version of the LazyLedger chain.                                                             |
+| `chainID`           | `uint64`                  | Chain ID. Each fork assigns itself a (unique) ID.                                            |
+| `height`            | `uint64`                  | Block height. The genesis block is at height `1`.                                            |
+| `time`              | [Time](#time)             | Timestamp of this block.                                                                     |
+| `lastBlockID`       | [BlockID](#blockid)       | Previous block's ID.                                                                         |
+| `lastCommitRoot`    | [HashDigest](#hashdigest) | Previous block's Tendermint commit root.                                                     |
+| `consensusHash`     | [HashDigest](#hashdigest) | Consensus parameters for this block.                                                         |
+| `stateCommitment`   | [HashDigest](#hashdigest) | Commitment to state root and validator set root after this block's transactions are applied. |
+| `availableDataRoot` | [HashDigest](#hashdigest) | Root of [commitments to erasure-coded data](#availabledataheader).                           |
+| `proposerAddress`   | [Address](#address)       | Address of this block's proposer.                                                            |
```

Comment:
![image](https://user-images.githubusercontent.com/7441766/78677351-edc34f80-78e7-11ea-9431-436a610021a1.png)

`lastBlockId` in the markdown text looks different? 🤔 

https://github.com/LazyLedger/lazyledger-specs/blob/adlerjohn-refactor_header_roots/specs/data_structures.md#blockid

Reply:
Oh yeah, that arrow should be from the `availableDataRoot`

Reply:
Fixed in 7c85201